src/main/java/model/Lotto.java

```diff
@@ -5,16 +5,39 @@
 
 public class Lotto {
     private List<Integer> lotto = new ArrayList<>();
+    private int bonus;
 
     public Lotto() {
     }
 
+    public Lotto(List<Integer> lotto, int bonus) {
+        this.lotto = lotto;
+        this.bonus = bonus;
+    }
+
     public Lotto(List<Integer> lotto) {
         this.lotto = lotto;
     }
 
     public List<Integer> getLotto() {
         return this.lotto;
     }
+  
+    public boolean matchBonus() {
+        if (this.lotto.contains(this.bonus))
+            return true;
+
+        return false;
+    }
+
+    public int matchCount(List<Integer> winLotto) {
```

Comment:
```suggestion
    public int matchCount(Lotto lotto) {
```

로또끼리 비교하는 것이 좋을 것 같아요!